### ISSUE TITLE
[CRASH] - Radio option crash app

### ISSUE THREAD

**Issue by black00019** [CappielloAntonio/tempo#214]
## Description
When click su radio artist app crash and close

## Environment
 - Android device: Pixel 7
 - Android OS version: 14
 - App version: 3.8.1
 - Other relevant details: [e.g., specific network conditions, external dependencies]

**Comment by black00019** [comment]
Risolto, colpa di Navidrome e Last.fm

**Comment by ramytarchichy** [comment]
Same issue here

### Steps to reproduce

Tap the "Radio" button in any artist's view

### Environment

- Android device: Pixel 7
- GrapheneOS version 14 (build 2024050700)
- App version: 3.8.1

**Comment by CappielloAntonio** [comment]
Hi @ramytarchichy, 
would you be able to capture the crash log with this apk?

[app-notquitemy-debug.zip](https://github.com/CappielloAntonio/tempo/files/15445064/app-notquitemy-debug.zip)

**Comment by lunarna-gh** [comment]
This is reproducible if you haven't favorited any songs by the artist. The mix and radio buttons work the same in my app as they both only cycle favorited songs, but the mix is grayed out if you have no favorites. I'm running navidrome, the phone is GrapheneOS on a Pixel 7, happens in both the build from F-Droid and the attached apk. @CappielloAntonio Where does the apk store the crash logs?

**Comment by lunarna-gh** [comment]
I found the issue, Navidrome needs to be linked to last.fm. There should probably be an error message if the server doesn't support the feature or a way to hide it

**Comment by asm0dey** [comment]
Yep, connecting last.fm helps. For it I had to create an account :)
